Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -132,4 +134,9 @@ struct FPTIBatchData: Codable {
             case tokenizationKey = "tokenization_key"
         }
     }
+
+    private static func isVenmoAppInstalled() -> Bool {
+        let venmoURL = URL(string: "com.venmo.touch.v2://")!
```

Comment:
Can we move this into the `BTCoreConstants` file? This way we can access it in here & both VenmoClient, and reduce risk of copy paste errors